tox.ini

```diff
@@ -1,4 +1,4 @@
 [tox]
-envlist = py27
+envlist = py27, py34
```

Comment:
I guess we can't specify something more generic here such as `py3`? Could we add `py35` and `py36` to this list in a subsequent pull request?

I suppose, more broadly, I'm wondering whether we're committing to supporting Python 3.4 specifically, Python 3 generically, or some range of Python 3 versions, such as Python 3.4+.

Reply:
I think adding `py35` and `py36` in the near future would be great.